converter/converter.go

```diff
@@ -71,12 +78,25 @@ func toYAML(jsonTemplate []byte) ([]byte, error) {
 
 func toJSON(yamlTemplate []byte) ([]byte, error) {
 	jsonTemplate, error := yaml.YAMLToJSON(yamlTemplate)
+	if !govalidator.IsJSON(string(jsonTemplate)) {
+		return nil, errors.New("This is not a valid YAML file")
+	}
+	return jsonTemplate, error
+}
+
+func prettyJSON(yamlTemplate []byte) ([]byte, error) {
+	var yamlObj interface{}
+	templateError := yaml.Unmarshal(yamlTemplate, &yamlObj)
+	if templateError != nil {
+		return nil, errors.New("Incorrect yaml")
```

Comment:
Let's use capitalization for _YAML_ here. Also, I think we should unify those error messages between other cases.